node-operators/network-node/tanssi/tanssi-node-docker.md

```diff
@@ -0,0 +1,146 @@
+
+=== "Dancelight"
+
+    ```bash
+    mkdir /var/lib/dancelight-data
+    ```
+
+Set the folder's ownership to the account that will run the Docker image to ensure writing permission:
+
+=== "Dancelight"
+
+    ```bash
+    chown INSERT_DOCKER_USER /var/lib/dancelight-data
+    ```
+
+Or run the following command if you want to run the node with the current logged-in user:
+
+=== "Dancelight"
+
+    ```bash
+    sudo chown -R $(id -u):$(id -g) /var/lib/dancelight-data
+    ```
+
+!!! note
+    The directory is a parameter in the Docker start-up command. If you decide to create the directory elsewhere, update the command accordingly.
+
+### Generate the Node Key {: #generate-node-key }
+
+To generate and store on disk the session keys that will be referenced on the start-up command, run the following command:
+
+=== "Dancelight"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancelight-data:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    {{ networks.dancelight.operator_docker_image }} key generate-node-key --file /data/node-key
+    ```
+
+--8<-- 'text/node-operators/sequencers/onboarding/run-a-sequencer/generate-node-key-unsafe-note.md'
+
+## Start Your Node {: #start-your-node }
+
+To spin up your node, you must run the Docker image with the `docker run` command.
+
+Replace `INSERT_YOUR_TANSSI_NODE_NAME` with a human-readable name and set `YOUR_IP_ADDRESS` with your public IP address.
+
+--8<-- 'text/node-operators/optimized-binaries-note.md'
+
+=== "Dancelight"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancelight-data:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    {{ networks.dancelight.operator_docker_image }} \
+    --chain dancelight \
+    --base-path /data \
+    --name INSERT_YOUR_TANSSI_NODE_NAME \
+    --node-key-file /data/node-key \
+    --rpc-port 9944 \
+    --prometheus-port 9615 \
+    --prometheus-external \
+    --listen-addr /ip4/0.0.0.0/tcp/30333 \
+    --public-addr /ip4/YOUR_IP_ADDRESS/tcp/30333 \
+    --state-pruning archive \
+    --blocks-pruning archive \
+    --database paritydb \
+    --unsafe-rpc-external \
+    --telemetry-url 'wss://telemetry.polkadot.io/submit/ 0'
+    ```
+
+### Run Flags {: #run-flags }
+
+The flags used in the `docker run` command can be adjusted according to your preferences and hardware configuration. The following ones are some of the most note-worthy:
+
+- `--state-pruning=archive` - Keeps all state data, which is necessary for historical state queries
+- `--blocks-pruning=archive` - Keeps all blocks, necessary for historical block data
+- `--database=paritydb` - Uses ParityDB as the database backend, which is optimized for RPC node performance
+- `--unsafe-rpc-external` - Allows external connections to the RPC server. This is required for the node to be accessible externally, but exposing RPC endpoints carries security risks. Ensure appropriate firewall and security measures are in place (see warning below)
```

Comment:
These should have the items on the left side in bold and descriptions on the right should not start with a capitalized letter. For example, the first item should be:

- **`--state-pruning=archive`** - keeps all state data, which is necessary for historical state queries